react/SquareAppIcon/index.jsx

```diff
@@ -109,12 +109,12 @@ export const SquareAppIcon = ({
           size="large"
           overlap="rectangular"
           style={
-            variant === 'shortcut'
+            variant === 'shortcut' && !IconContent
               ? { backgroundColor: nameToColor(name) }
               : null
           }
         >
-          {variant === 'shortcut' ? (
+          {variant === 'shortcut' && !IconContent ? (
```

Comment:
nit: let's add a little test to prevent this regression to happen again.

I am preparing the commit

Reply:
done

Reply:
To me, this is the argos' job to check we don't have any regression. 

Here, even with this test, we would have the regression, why? Because we created a brand new component with a new behavior. But we forgot to add one case. 

So neither UT or Argos will save us in that case. The only thing that can save us is to do TDD on that kind of development